packages/react-composites/src/composites/CallWithChatComposite/ChatAndPeoplePane.tsx

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { concatStyleSets, DefaultButton, Stack } from '@fluentui/react';
+import { ChevronLeft28Regular } from '@fluentui/react-icons';
+import { ParticipantList, useTheme } from '@internal/react-components';
+import React, { useMemo } from 'react';
+import { CallAdapter } from '../CallComposite';
+import { CallAdapterProvider } from '../CallComposite/adapter/CallAdapterProvider';
+import { usePropsFor } from '../CallComposite/hooks/usePropsFor';
+import { ChatAdapter, ChatComposite } from '../ChatComposite';
+import { AvatarPersonaDataCallback } from '../common/AvatarPersona';
+import { ParticipantListWithHeading } from '../common/ParticipantContainer';
+import {
+  chatAndPeoplePaneButtonStyles,
+  chatAndPeoplePaneCloseButtonStyles,
+  chatAndPeoplePaneContentHiddenStyle,
+  chatAndPeoplePaneContentStyle,
+  chatAndPeoplePaneControlBarStyle,
+  chatAndPeoplePaneHiddenStyle,
+  chatAndPeoplePaneStyle
+} from './styles/ChatAndPeoplePaneStyles';
+import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
+import { removeParticipantFromCallWithChat } from './SidePane';
+
+export const ChatAndPeoplePane = (props: {
+  chatAdapter: ChatAdapter;
+  callAdapter: CallAdapter;
+  closePane: () => void;
+  onChatButtonClicked: () => void;
+  onPeopleButtonClicked: () => void;
+  activeTab?: ChatAndPeoplePaneTab;
+  onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+}): JSX.Element => {
+  const {
+    chatAdapter,
+    callAdapter,
+    closePane,
+    onChatButtonClicked,
+    onPeopleButtonClicked,
+    onFetchAvatarPersonaData,
+    activeTab
+  } = props;
+
+  const theme = useTheme();
+  const strings = useCallWithChatCompositeStrings();
+  const chatAndPeoplePaneButtonStylesThemed = concatStyleSets(chatAndPeoplePaneButtonStyles, {
+    rootChecked: {
+      borderBottom: `0.125rem solid ${theme.palette.themePrimary}`
+    }
+  });
+
+  return (
+    <Stack verticalFill grow className={activeTab ? chatAndPeoplePaneStyle : chatAndPeoplePaneHiddenStyle}>
```

Comment:
For these different components could we make them different files? Could be stylistic but these files are getting pretty big and we are just changing what we are passing in?